### PR TITLE
Do not validate VM for any OX Value Representation

### LINKED ISSUE #1186: Breaking change around FloatPixelData between 4.0.1 and 4.0.2
**Describe the bug**
The following sample code that adds FloatPixelData to a DicomDataset works fine with 4.0.1, but throws an exception with 4.0.2. 
```           
DicomDataset dicomDataset = new DicomDataset
{
    { DicomTag.Rows, (ushort)1 },
    { DicomTag.Columns, (ushort)2 },
};

dicomDataset.AddOrUpdate(new DicomDataset { { DicomTag.FloatPixelData, new float[] { 1.0f, 2.0f } } });
```

Exception thrown:
> 'Content "(7fe0,0008) OF Float Pixel Data" does not validate VR OF: Number of items 2 does not match ValueMultiplicity 1'

**To Reproduce**
Run the above snippet using fo-dicom 4.0.1. 
Verify no exceptions are thrown.
Update fo-dicom from 4.0.1 to 4.0.2.
Verify an exception is thrown.

**Expected behavior**
No exceptions and the DicomDataset is properly create with FloatPixelData.

**What I tried**
I tried disabling auto-validate but it didn't have any effects:
`dicomDataset.AutoValidate = false;`

**Environment**
Fellow Oak DICOM version: 4.0.1 and above
OS: Windows 10 x64
Platform: .NET Core 5.0

### PR DESCRIPTION
**Pull request by mrbean-bremen**
- disable check for OL, OF and OV (the rest is already handled)
- fixes #1186

#### Checklist
- [x] The pull request branch is in sync with latest commit on the *fo-dicom/development* branch
- [ ] I have updated API documentation
- [x] I have included unit tests
- [x] I have updated the change log
- [x] I am listed in the CONTRIBUTORS file

Also added cases for VRs UV, SV and OV  in DicomDataset.DoAdd() (needed OV in the unit test).

### CODE REVIEW

**Comment by mrbean-bremen** [comment]
Note that this is only done for fo-dicom 5, as I don't know the policy for fixes in version 4. Are they also branched from the development branch? 
If needed I can add the same changes to the old classes (the changes are very trivial, so this would be a no-brainer).

**Comment by gofal** [comment]
Thanks for this pull request!

The policy is that enhancements are mainly done in version 5. Bugfixes and small enhancements are OK in version 4, but have to be also done in version 5.
The issue was reported in version 4, so it is fine to fix it in version 4, too.

**Comment by mrbean-bremen** [comment]
Thansk - I will add that in version 4 too, in this case.

File: Tests/Desktop/DicomValidationTest.cs [hunk 1]
```diff
@@ -158,57 +173,57 @@ public void ValidateLongText()
...
 
-Duis autem vel eum iriure dolor in hendrerit in vulputate velit esse molestie consequat, vel illum dolore eu feugiat nulla facilisis at vero eros et accumsan et iusto odio dignissim qui blandit praesent luptatum zzril delenit augue duis dolore te feugait nulla facilisi. Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat. 
+Duis autem vel eum iriure dolor in hendrerit in vulputate velit esse molestie consequat, vel illum dolore eu feugiat nulla facilisis at vero eros et accumsan et iusto odio dignissim qui blandit praesent luptatum zzril delenit augue duis dolore te feugait nulla facilisi. Lorem ipsum dolor sit amet, consectetuer adipiscing elit, sed diam nonummy nibh euismod tincidunt ut laoreet dolore magna aliquam erat volutpat.
```
Comment (mrbean-bremen) [inline Tests/Desktop/DicomValidationTest.cs:226]: This change was unintended - I seem to have used some auto-formatting that removed trailing spaces.

**Comment by mrbean-bremen** [comment]
@gofal - is there anything left to do here? I'm not sure about the PR workflow in fo-dicom...

**Comment by mrbean-bremen** [comment]
The other reason why I'm interested in finishing this PR is to have the possibility to see the CI output for other PRs. As long as as I haven't done any contribution, I have to wait for each action workflow run to be approved before it is actually run - I just like to check if the tests passed in the CI the same day I did the change 😁

**Comment by gofal** [comment]
everything seems fine here. Usually PRs are open for some time to allow other contributors to review or add comments - not everyone is on github daily. Smaller PRs can me merged sooner, of course. I guess I will merge this tomorrow.

**Pull request merged**